v2-migration/src/test/java/software/amazon/awssdk/v2migration/EnumCasingToV2Test.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Can we add some end-to-end tests? https://github.com/aws/aws-sdk-java-v2/tree/feature/master/migration-tool/test/v2-migration-tests